package.json

```diff
@@ -94,6 +98,7 @@
     "@nrwl/nx-cloud": "13.1.2",
     "@nrwl/tao": "12.10.1",
     "@nrwl/workspace": "12.10.1",
+    "@nx-plus/docusaurus": "12.2.0",
```

Comment:
Used to generate a Docusaurus project in an Nx workspace as well as offer `build` and `serve` targets. Also includes migrations.